packages/lib/passport-interface/src/SubscriptionManager.ts

```diff
@@ -182,11 +192,14 @@ export class FeedSubscriptionManager {
    */
   public async pollSubscriptions(
     credentialManager: CredentialManagerAPI,
-    onFinish?: (actions: SubscriptionActions) => Promise<void>
+    onFinish?: (actions: SubscriptionActions) => Promise<void>,
+    idsToPoll?: Array<string | undefined>
```

Comment:
Given the usage to filter out the email subscription, it might make sense to make that an explicit feature here, or else have this arg be a list of IDs to skip, which would be a fixed list rather than the caller needing to do filtering.
Or maybe do both and have a whitelist+blacklist approach.

Reply:
The reason I did this is that the `passport-interface` package doesn't know which feed is the email feed - that's known in `passport-client`. i thought about several ways to do this and this seemed the least onerous.